tools/shared.py

```diff
@@ -207,7 +207,9 @@ def new(*args):
     EM_CONFIG = EM_CONFIG.replace(';', '\n') + '\n'
 
 if not EM_CONFIG:
-  EM_CONFIG = '~/.emscripten'
+  EM_CONFIG = path_from_root('emscripten.rc')
+  if not os.path.exists(EM_CONFIG):
+    EM_CONFIG = '~/.emscripten'
```

Comment:
In the future, perhaps it would be nice if the user config could inherit from the root config (if it exists).